src/pages/registration/resource_type_tab/sub_type_forms/artistic_types/music_performance/ArtisticMusicPerformanceForm.tsx

```diff
@@ -98,27 +99,31 @@ export const ArtisticMusicPerformanceForm = () => {
 
               <Box sx={{ mt: '1rem', display: 'flex', flexDirection: { xs: 'column', sm: 'row' }, gap: '1rem' }}>
                 <Button
+                  sx={{ textTransform: 'none' }}
```

Comment:
Her er det eksplisitt tatt vekk uppercase mens det fortsatt er uppercase for andre typer 🤔

![Image](https://github.com/user-attachments/assets/4e07a78a-8687-4b4e-9e4a-ff80d3733c49)
vs
![Image](https://github.com/user-attachments/assets/07cf17ba-b657-4f5b-850f-2891eb1d2aa8)

Ser vi tenderer til å opte-ut fra uppercase som er default i MUI i flere tilfeller, så mulig dette burde endres i theme snart også, da jeg ikke ser noen god grunn til at vi vil at noen knapper skal være uppercase men andre ikke. Men det er en annen sak. Bare sørg for at de blir like her hvertfall, uansett hva det er.